lib/src/config/config_file/client.example.toml

```diff
@@ -19,15 +19,10 @@
 # Multiple addresses can be specified.
 #
 # If `advertised_addresses` is not used, these addresses will also be advertised to peers.
-listen_addresses = [
-  "/ip4/0.0.0.0/tcp/8443/ws",
-  "/ip6/::/tcp/8443/ws",
-]
+listen_addresses = ["/ip4/0.0.0.0/tcp/8443/ws", "/ip6/::/tcp/8443/ws"]
 
 # Seed nodes to connect to when the node starts up. Multiple addresses can be specified.
-seed_nodes = [
-  { address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" },
-]
+seed_nodes = [{ address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" }]
```

Comment:
Please don't change this formatting.

Reply:
I put it back to the previous formating.